.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - django-1.10
+      - django-2.0
+
+jobs:
+  django-1.10:
```

Comment:
I think you might be able to factorize that quite extensively with some YAML anchors. :-)
The format is definitely confusing, but [our config for `facebook-python-ads-sdk`](https://github.com/Work4Labs/facebook-python-ads-sdk/blob/ce1bddcc49980879f9d2144cf82dddedb70f2fce/.circleci/config.yml) might help. Maybe put the Django version in an ENVVAR while factorizing?

Reply:
✅